bl/validation/k8sValidator.go

```diff
@@ -124,7 +125,8 @@ func (val *K8sValidator) validateResource(filepath string) (bool, []error, error
 		// File starts with ---, the parser assumes a first empty resource
 		if res.Status == kubeconformValidator.Invalid || res.Status == kubeconformValidator.Error {
 			isValid = false
-			validationErrors = append(validationErrors, &InvalidK8sSchemaError{ErrorMessage: res.Err.Error()})
+			errorMessage := strings.ReplaceAll(res.Err.Error(), "-", "\n   ")
```

Comment:
I think it would be better to split this string into different "messages" and add each message as an element to the `validationErrors` array